projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/JobExecutionFilterSpec.java

```diff
@@ -24,11 +24,16 @@
 public class JobExecutionFilterSpec implements Specification<DataJobExecution> {
 
    private DataJobExecutionFilter filter;
+   private String jobName;
 
    @Override
    public Predicate toPredicate(Root<DataJobExecution> root, CriteriaQuery<?> query, CriteriaBuilder builder) {
       List<Predicate> predicates = new ArrayList<>();
 
+      if (jobName != null) {
```

Comment:
I wonder if we could reuse filter.getJobNameIn()?

Reply:
That was my initial consideration, but I decided against it for simplicity's sake. Using the getJobNameIn for this purpose would require changes to the filter for each individual data job. Now the implementation is much simpler.

Reply:
The method behavior is a bit strange to me. If you set both _jobName_ and _jobNameIn_ the query will look like this:
```
SELECT * FROM data_job_execution WHERE job_name = "A" and job_name IN ['B', 'C']
```
`Using the getJobNameIn for this purpose would require changes to the filter for each individual data job. ` - what do you mean by `require changes to the filter for each individual data job` isn't it just adding an job name to the _jobNameIn_ list?

Reply:
I will refactor to reuse `jobNameIn`.